lm_eval/evaluator.py

```diff
@@ -213,7 +213,10 @@ def evaluate(
         # aggregate Instances by LM method requested to get output.
         reqtype = (
             "loglikelihood"
-            if (task.OUTPUT_TYPE == "multiple_choice" or task.OUTPUT_TYPE == "winograd_schema") 
+            if (
+                task.OUTPUT_TYPE == "multiple_choice"
+                or task.OUTPUT_TYPE == "winograd_schema"
```

Comment:
we're now removing `winograd_schema` right?

Reply:
Yes, but wasn't sure to remove it in this PR. Do you think it's okay?

Reply:
I think so!